style-guides/python.md

```diff
@@ -0,0 +1,32 @@
+# Python Infrastructure and Coding Standards 
+
+Here at Systers open source we respect the [PEP8 style guide for Python](http://www.python.org/dev/peps/pep-0008/). Docstrings follow [PEP257](http://www.python.org/dev/peps/pep-0257/). In addition to the guidelines specified by PEP8 we have organizatin specific guidelines and various tools we use to enforce these guidelines. We want to make sure all our python projects are in synchrony with these guidelines.
+
+The last of this document talks about the tools we use for automated reviews, CI and CD online.
+
+ 
+
+## PEP8
+
+Please refer to the [PEP8 Document](http://www.python.org/dev/peps/pep-0008/) to and find out about rules concerning indentation, use of tabs or spaces etc). We use flake8 to enfore these rules. Please bear in mind that [**flake8**]() may not include everything in the PEP8 guide and contains other checks not enforced by the guide as it's basically a combination of `PyFlakes`, `pycodestyle` (PEP8 intensive) and `Ned Batchelder’s McCabe script`
```

Comment:
extra `)` after etc?

Reply:
I think we should also document a common `setup.cfg` for flake8 so every repo uses the same config?

Reply:
Could you do that? or mock up for me to see?

Reply:
Yeah, sure I'll do it. :) shall I add it in this PR itself? as `standard-setup.cfg`?